ui/components/reservation-unit/QuickReservation.tsx

```diff
@@ -291,7 +288,7 @@ const QuickReservation = ({
     const [hours, minutes] =
       duration?.value.toString().split(":").map(Number) || [];
     const length = hours * 60 + minutes;
-    return getReservationUnitPrice(reservationUnit, date, length);
+    return getReservationUnitPrice(reservationUnit, date, length, true);
```

Comment:
Näissä "magic variable" muutoksissa ois kyl kiva että nämä funktiot, joilla on enemmän kun 1 argumentti ois tyypitettyjä. tuostaki tietäis mihin toi "true" oikeesti liittyy, navigoimatta funktioon.

Reply:
Jep, ihan asiasta jupiset. Pidetään korvan takana - nyt speksin säätö soi ajan.